exercises/5.3/template_ws/ros2/src/filter_call/filter_call/filter_call.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+import rclpy
+from rclpy.node import Node
+import py_perception.srv
+from py_perception.srv import FilterCloud
+from sensor_msgs.msg import PointCloud2
+
+class FilterNode(Node):
+    def __init__(self):
+        super().__init__('filter_cloud', allow_undeclared_parameters = False,
+                         automatically_declare_parameters_from_overrides = True)
```

Comment:
As above, so below.  Why is allow_undeclared_parameters true in C++ but false in Python?